es-app/src/SystemData.cpp

```diff
@@ -112,8 +112,13 @@ void SystemData::populateFolder(FileData* folder)
 				continue;
 
 			FileData* newGame = new FileData(GAME, filePath, mEnvData, this);
-			folder->addChild(newGame);
-			isGame = true;
+
+			// preventing new arcade assets to be added
+			if(!newGame->isArcadeAsset())
```

Comment:
I can confirm it goes through here, but doesn't get added here.

Reply:
Just to answer your comment: here is the place where files not in `gamelist.xml` are added.